* Add 1000 to LINUX_API version. Avoids crashes between branches devel and cofb.

--- linux-2.6.26.8-source.orig/include/linux/cooperative.h
+++ linux-2.6.26.8-source/include/linux/cooperative.h
@@ -20,7 +20,7 @@
 
 #include <asm/cooperative.h>
 
-#define CO_LINUX_API_VERSION    14
+#define CO_LINUX_API_VERSION    1014
 
 #pragma pack(0)
 
 
